safe_transaction_service/history/indexers/ethereum_indexer.py

```diff
@@ -474,40 +483,24 @@ def start(self) -> Tuple[int, int]:
                 "%s: No almost updated addresses to process", self.__class__.__name__
             )
 
-        not_updated_addresses = list(
-            self.get_not_updated_addresses(current_block_number)
-        )
-        if not_updated_addresses:
+        # Then process addresses that are not updated (usually far from the `current_block_number`)
+        if not_updated_addresses := self.get_not_updated_addresses(
+            current_block_number
+        ):
             logger.info(
-                "%s: Processing %d not updated addresses total",
+                "%s: Processing not updated addresses",
                 self.__class__.__name__,
-                len(not_updated_addresses),
             )
 
-            # Not updated addresses are sorted by tx_block_number
```

Comment:
This is not True now, are you aware of this?

Reply:
I'm aware, it shouldn't be relevant now, but thanks for reminding me :)